src/components/Gradebook/index.jsx

```diff
@@ -463,6 +480,13 @@ export default class Gradebook extends React.Component {
         },
       );
       this.updateQueryParams({ courseGradeMin, courseGradeMax });
+      trackEvent(() => ({
```

Comment:
Previously we only had actions which fired off in tandem with a particular redux action was dispatched. I believe we can use the same pattern here, since `updateCourseGradeFilter` does fire a redux action.